docs/en/journal.md

```diff
@@ -108,6 +108,22 @@ The parameter id in the route is the id in the URL of the user ’s Google Schol
 
 </RouteEn>
 
+## IEEE Xplore
+
+### Latest Research
+
+<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name， get it from punumber of the URL','Sort Type, default: `vol-only-seq`，get it from sortType of the URL']" radar="1" rssbud="1">
+
+</RouteEn>
+
+### Latest Research (Last 2 month)
+
+<RouteEn author="Derekmini auto-bot-ty" example="/ieee/78/latest/date" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name， get it from punumber of the URL','Sort Type, default: `vol-only-seq`，get it from sortType of the URL']" radar="1" rssbud="1">
```

Comment:
```suggestion
<RouteEn author="Derekmini auto-bot-ty" example="/ieee/78/latest/date" path="/ieee/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
```

Reply:
Thanks your attention, I have modified it.